website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -610,8 +610,15 @@ Transfer-Encoding: chunked
 ## Feature Variants
 
 ### Put variants for Feature Toggle {#update-variants}
+:::caution 
 
-<ApiRequest verb="put" url="api/admin/projects/:projectId/features/:featureName/variants" title="Create (overwrite) variants for a feature toggle (example data)" payload={[
+From 4.21 variants are tied to an environment. Check our Open API docs for the up-to-date docs: https://docs.getunleash.io/reference/api/unleash/features
```

Comment:
Same question here regarding the link? Why are we linking them there?

```suggestion
Since Unleash, 4.21 variants are tied to environments.
```

Reply:
Got it. So as above: it should be linking the the openapi doc for this specific endpoint, right? `/docs/reference/api/unleash/overwrite-feature-variants.api.mdx`? I'll make a suggestion for that.

Reply:
yes, I think that'd be the best to start making it our source of truth